sample.env

```diff
@@ -14,6 +14,9 @@ HEADER_X_API_KEY=
 KAFKA_TYPE=
 CONFLUENT_KEY=
 CONFLUENT_SECRET=
+KAFKA_ENABLE_AUTO_COMMIT=
+KAFKA_AUTO_COMMIT_INTERVAL_MS=
+KAFKA_SESSION_TIMEOUT_MS=
```

Comment:
It might be good to put a comment in here saying that if these are not set, they will have default values.

Reply:
Updated this to include comments with the default values for each of these variables.

Reply:
I'm unsure whether comments are allowed on the same line as environment variables in .env files. Developers might need to delete the comments to set the environment variables correctly. It's probably better to have these comments on separate lines from the environment variables.

Reply:
Updated to have comments on separate lines